nio/schemas.py

```diff
@@ -1213,6 +1213,32 @@ class Schemas(object):
         ],
     }
 
+    tags = {
+        "type": "object",
+        "properties": {
+            "type": {"type": "string"},
+            "content": {
+                "type": "object",
+                "properties": {
+                    "tags": {"type:": "object"},
+                    "properties": {
+                        "order": {"type": "float"}
```

Comment:
The float type isn't a valid jsonschema type, you'll have to use numeric here. You can read up on this [here](https://json-schema.org/understanding-json-schema/reference/numeric.html#number)